apps/google-analytics-4/lambda/src/controllers/apiController.ts

```diff
@@ -44,7 +44,7 @@ const ApiController = {
         serviceAccountKeyFile
       );
 
-      return res.send(200);
+      return res.sendStatus(200);
```

Comment:
Just curious if we want to be sending the string representation of the HTTP status code, or if we want to maintain the actual code?

sends status code
```typescript
res.status(200);
```

sends string representation of status code
```typescript
res.sendStatus(200);
```

Reply:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/1434735/230459398-181484bc-75cc-4d0e-a9b6-3e837bd0af81.png">
I think it's just a number?

Reply:
<img width="510" alt="image" src="https://user-images.githubusercontent.com/1434735/230459518-9363bd3f-6d9d-4bb5-9495-30597280e61c.png">

Reply:
interesting, so short hand for the chainable 🤔 👍 neato! Just curious 🙇